api/hydra/api.ttl

```diff
@@ -591,6 +592,23 @@ api:AddAttributeMappedFromSingleColumn
     )
   ] .
 
+_:PostReferenceAttribute
+  a hydra:SupportedOperation, hydra-box:View, api:AddReferenceAttribute ;
+  hydra:method "POST" ;
+  hydra:title "Creates a reference attribute" ;
+  hydra:description "Creates a table attribute referencing a dimension table" ;
+  hydra:expects dataCube:ReferenceAttribute ;
```

Comment:
I can't find a description of `ReferenceAttribute` in the file. Is it missing or not required?

Reply:
Ah my mistake. Of course, will add it